index.html

```diff
@@ -1,12 +1,233 @@
+      <p class="advantage-reasons">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+      <div class="advantage advantage-blue advantage-first">
+        <h3 class="advantage-title">
+          НАСТОЯЩИЙ <br> ГОРОДОК
+        </h3>
+        <span class="advantage-nomber">
+          - №1 -
+        </span>
+        <p class="advantage-description">
+          Седона - не аттракцион <br>
+          для туристов, там течёт <br>
+          своя жизнь
+        </p>
+      </div>
+      <div class="advantage-pic-first">
+        <img src="" alt="Фотография города">
+      </div>
+      <ul class="advantage-list">
+        <li class="advantage advantage-item-dark-grey">
+          <img class="advantage-title-pic" src="" alt="Дом">
+          <h3 class="advantage-title">
+            ЖИЛЬЁ
+          </h3>
+          <p class="advantage-description">
+            Рекомендуем пожить <br>
+            в настоящем мотеле, <br>
+            всё как в кино!
+          </p>
+        </li>
+        <li class="advantage advantage-item-white">
+          <img class="advantage-title-pic" src="" alt="Бургер">
+          <h3 class="advantage-title">
+            ЕДА
+          </h3>
+          <p class="advantage-description">
+            Всегда заказывайте <br>
+            топовый фирменный бургер, <br>
+            вы не разочаруетесь!
+          </p>
+        </li>
+        <li class="advantage advantage-item-dark-grey">
+          <img class="advantage-title-pic" src="" alt="Подарочная коробка с лентой">
+          <h3 class="advantage-title">
+            Сувениры
+          </h3>
+          <p class="advantage-description">
+            Не только китайского <br>
+            но и настоящего местного <br>
+            производства!
+          </p>
+        </li>
+      </ul>
+      <div class="advantage-pic-second">
+        <img src="" alt="Фотография моста Дьявола">
+      </div>
+      <div class="advantage advantage-blue advantage-second">
+        <h3 class="advantage-title">
+          ТАМ ЕСТЬ <br> МОСТ ДЬЯВОЛА
+        </h3>
+        <span class="advantage-nomber">
+          - №2 -
+        </span>
+        <p class="advantage-description">
+          Да, по нему можно пройти!<br>
+          Если вы осмелитесь,<br>
+          разумеется
+        </p>
+      </div>
+      <ul class="advantage-list">
+        <li class="advantage advantage-item-white-grey">
+          <h3 class="advantage-title">
+            НЕБОЛЬШАЯ <br> ПЛОЩАДЬ
+          </h3>
+          <span class="advantage-nomber">
+            - №3 -
+          </span>
+          <p class="advantage-description">
+            Все достопримечательности <br>
+            находятся очень близко
+          </p>
+        </li>
+        <li class="advantage advantage-item-blue-gray">
+          <h3 class="advantage-title">
+            КРАСИВАЯ <br> ДОРОГА
+          </h3>
+          <span class="advantage-nomber">
```

Comment:
```suggestion
          <span class="advantage-number">
```

если я там пропустила, везде намберы поправь :) Номбер - это вроде "человек" на итальянском. Звучит похоже, по крайней мере :)